src/components/TakeCareContainer/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import CareCardPrevention from '../CareCardPrevention/index';
+import CareCards from '../CareCards/index';
+import './TakeCareContainer.scss';
+
+const TakeCareContainer = () => {
+  return (
+    <React.Fragment>
+      <section className='TakeCare__main'>
+        <h4 className='TakeCare__title'>
```

Comment:
In the components is a `TitleSection` component you can use it here and can fix the color in the styles.
This component receives two props: title and infoButton if the title does not include info button you can only pass the title.